evo1/atb.py

```diff
@@ -165,10 +187,10 @@ def predict_damage(self, attacker: BattleEntity, defender: BattleEntity) -> int:
 
     def predict_hit(self, attacker: BattleEntity, defender: BattleEntity) -> bool:
         rng = EvolandRNG().get_rng()
-        accuracy = 10 # TODO: from attacker
-        evade = 10 # TODO: from defender
-        miss_chance = 30 # TODO: Calc
-        roll = (rng.rand_int() & 0x3fffffff) % 100
+        # accuracy = 10  # TODO: from attacker
+        # evade = 10  # TODO: from defender
```

Comment:
This entire function is unused until I implement it properly (I need to find the correct algorithm/memory to read first). But commenting it out for now is fine.